mod-patron-blocks/src/main/resources/domain/mod-patron-blocks/features/samples/item-entity.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This item record looks incomplete. First thing that catches my attention - missing `holdingsRecordId`.